### ISSUE TITLE
Bottom nav is not aria-hidden after switching to scrolling mode

### ISSUE THREAD

**Issue by aslagle** [NYPL-Simplified/webpub-viewer#115]
When you open the settings and switch to scrolling mode, the bottom nav should be displayed but not focusable by screen readers, since the settings is still open.
